apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -126,4 +184,39 @@ DropletAutocompleteParameterTooltipManager.prototype.getTooltipHTML = function (
   });
 };
 
+/**
+ * @param editor - ace editor instance
+ * @private
+ */
+DropletAutocompleteParameterTooltipManager.installAceCompleterHooks_ = function (editor) {
+  if (editor.completer.showPopup !== DropletAutocompleteParameterTooltipManager.showPopup) {
+    DropletAutocompleteParameterTooltipManager.originalShowPopup = editor.completer.showPopup;
+    editor.completer.showPopup = DropletAutocompleteParameterTooltipManager.showPopup;
+  }
+  if (editor.completer.gatherCompletions !== DropletAutocompleteParameterTooltipManager.gatherCompletions) {
+    DropletAutocompleteParameterTooltipManager.originalGatherCompletions = editor.completer.gatherCompletions;
+    editor.completer.gatherCompletions = DropletAutocompleteParameterTooltipManager.gatherCompletions;
+  }
+};
+
+DropletAutocompleteParameterTooltipManager.gatherCompletions = function (editor, callback) {
+  // Override normal ace AutoComplete behavior by using only overrideCompleter
+  // instead of the normal set of completers when overrideCompleter is set
+  if (this.overrideCompleter) {
+    var allCompleters = editor.completers;
+    editor.completers = [ this.overrideCompleter ];
+    DropletAutocompleteParameterTooltipManager.originalGatherCompletions.call(this, editor, callback);
+    editor.completers = allCompleters;
+  } else {
+    DropletAutocompleteParameterTooltipManager.originalGatherCompletions.call(this, editor, callback);
+  }
+};
+
+DropletAutocompleteParameterTooltipManager.showPopup = function (editor) {
+  // Override normal ace AutoComplete behavior by guaranteeing that overrideCompleter is reset
```

Comment:
I meant to say that this function (showPopup) is overriding the normal behavior of `showPopup`.